main.go

```diff
@@ -122,6 +123,19 @@ func main() {
 		requireLogin.RequireLogin(home.HomeHandler),
 	)
 
+	// Export db page records to Alglolia index
+	log.Println("Exporting records to Algolia servers...")
+	pages, err := databaseActions.GetPages()
```

Comment:
I think we should only retrieve all pages if we determine that there isn't an index. Maybe algoliaUtil.ExportPageRecords could be modified to take a function which will retrieve the pages.